src/plugins/export/pdf.js

```diff
@@ -1,6 +1,6 @@
 import jsPDF from 'jspdf';
```

Comment:
don't import this files directly, use global namespace for them, also add them to vender list in webpack config.
now they are written to dist, also in documention please add that user should import this vendor libraries